tenant/src/main/protobuf/com/improving/app/tenant/domain/tenantTypeDefinitions.proto

```diff
@@ -2,33 +2,60 @@ syntax = "proto3";
 
 package com.improving.app.tenant.domain;
 import "scalapb/scalapb.proto";
+import "scalapb/validate.proto";
 import "google/protobuf/timestamp.proto";
 import "com/improving/app/common/domain/address.proto";
 import "com/improving/app/common/domain/ids.proto";
+import "validate/validate.proto";
+
+option (scalapb.options) = {
+  scope: FILE
+[scalapb.validate.file] {
+        validate_at_construction: true
+}
+        field_transformations: [
+        {
+                when: {options: {[validate.rules] {message: {required: true}}}}
+                set: {
+                [scalapb.field] {
+                required: true
+        }
```

Comment:
Something is off with the formatting here

Reply:
I have to format myself because IntelliJ isn't behaving well 
<img width="570" alt="Screenshot 2023-05-12 at 1 36 23 PM" src="https://github.com/improving-app/back-end/assets/105220856/9eb2a00c-041f-4d56-ab3d-90c22b72d438">
<img width="449" alt="Screenshot 2023-05-12 at 1 36 32 PM" src="https://github.com/improving-app/back-end/assets/105220856/f370bc08-aaff-4b7d-8d44-5087821f840a">

Reply:
When I copy-paste it goes all over the place
<img width="681" alt="Screenshot 2023-05-12 at 1 37 25 PM" src="https://github.com/improving-app/back-end/assets/105220856/87282e52-7755-4e00-8d1a-daaf4edbccdc">

Reply:
![INTELLIJ KHAN MEME](https://github.com/improving-app/back-end/assets/127801986/a514c890-5424-4533-adfc-53929326b13e)